### PR TITLE
Documentation for Kubelet Resource Metrics Endpoint

### PR DESCRIPTION
**Pull request by dashpole**
Documentation for https://github.com/kubernetes/enhancements/issues/727.  Apparrently we don't have any monitoring documentation, so this will be more broad than just the new kubelet endpoint.

/milestone 1.14

### CODE REVIEW

**Comment by dashpole** [comment]
This is ready for review

File: content/en/docs/concepts/cluster-administration/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+reviewers:
+- dashpole
+title: Monitoring Architecture
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+
+TODO Overview
```
Comment (jimangel) [inline content/en/docs/concepts/cluster-administration/monitoring.md]: Can you fix all of the TODO's on this page or remove the doc entirely?

Comment (dashpole) [reply content/en/docs/concepts/cluster-administration/monitoring.md]: whoops, that wasn't meant to be included.  It is removed

**Pull request merged**